docs/_static/custom.css

```diff
@@ -52,6 +52,6 @@ div.admonition.admonition-youtube > .admonition-title:before {
 }
 div.admonition.admonition-youtube > .admonition-title:after {
   color: #ff0000;
-  content: "\f26c"; /* fa-brands fa-tv */
+  content: "\f26c"; /* fa-solid fa-tv */
```

Comment:
@12rambau any reason you picked `f26c` here instead of [f167](https://fontawesome.com/icons/youtube?s=&f=brands)

Reply:
yes, the youtube icon is a brand icon, if you want to use it you also need to customize the font-family from "Font Awesome 6 Free" to "Font Awesome 6 Brands".

Reply:
Ah ok, so the brand icons don't work easily in an admonition? I didn't realize that.

Reply:
nope it doesn't, I saw it, tried to find an easy fix and realize we need to update fa font management first so I keep it in my mind and if someone complains about it I'll work on it but that's low priority I think